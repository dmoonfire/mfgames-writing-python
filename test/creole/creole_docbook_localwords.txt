= Testing LocalWords

The output of this file will not include the LocalWords paragraphs
below, but will include this paragraph.

LocalWords: LocalWords asdf askdfj lkgj dslkjsdf lkasjdf lksjdf
LocalWords: lkgjsldkjg ldfkjeobij sdvklaj sdflkj sdflkjsdf
