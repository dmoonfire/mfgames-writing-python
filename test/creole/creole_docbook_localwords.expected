<?xml version="1.0" encoding="UTF-8"?><article xmlns='http://docbook.org/ns/docbook' version="5.0"><info><title>Testing LocalWords</title></info><simpara>The output of this file will not include the LocalWords paragraphs below, but will include this paragraph.</simpara></article>